frontend/src/buttons/ComboButton/ComboButton.tsx

```diff
@@ -24,16 +24,22 @@ export const ComboButton: React.FC<Props> = ({ className, ...props }) => {
       {props.service?.attributes.route === 'p2p' && props.connection?.public ? (
         <div>
           <Notice fullWidth severity="info" gutterBottom>
-            {props.size === 'small'
-              ? 'Peer to peer only'
-              : 'You cannot make a proxy connection to this service, it is set to peer to peer only.'}
+            {props.size === 'small' ? (
+              'Peer to peer only'
+            ) : (
+              <>
+                You cannot make a proxy connection to this service, it is set to peer to peer only.
+                {!isPortal() && <i> Please try again in a few minutes.</i>}
+              </>
+            )}
           </Notice>
-          <DynamicButton
-            {...props}
-            title="Launch Desktop"
-            color="grayDarkest"
-            onClick={() => windowOpen(`${PROTOCOL}connect/${props.service?.id || props.connection?.id}`)}
-          />
+          {isPortal() && (
```

Comment:
Does this mean that the button text for Desktop doesn't change? Does it just look like the device is offline?

Reply:
Answer is in the slack thread here. https://remoteit.slack.com/archives/C019MLSJYS1/p1651765291970749